### PR TITLE
Feature/bump dependencies

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
**What does this PR solve?**
This PR solves the need to remove `Jcenter` since it stops it support at the end of March 2022, this fix comes to solve the following issue: https://github.com/cloudinary/cloudinary_android/issues/122

### Root cause
The root cause for this PR is the need to remove dependency of the SDK on the `Jcenter` repo.

Additional features
We increased the minimal Java version to 8
We increased the minimal Android SDK to 19

#### What does this PR address?
- [x] GitHub issue (Add reference - #XX)
- [x] Refactoring
- [ ] New feature
- [ ] Bug fix
- [] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,17 +22,22 @@ android:
...
     - extra-google-google_play_services
     - extra-google-m2repository
     - extra-android-m2repository
     - addon-google_apis-google-25
     - sys-img-armeabi-v7a-android-21
 
 before_install:
-  - yes | sdkmanager "platforms;android-28"
+  - touch $HOME/.android/repositories.cfg
+  - wget "https://dl.google.com/android/repository/commandlinetools-linux-7302050_latest.zip" -O commandlinetools.zip
```
Comment (const-cloudinary) [inline .travis.yml:37]: isn't there an easier way to install these tools?

File: sample/src/main/java/com/cloudinary/android/sample/app/ImageActivity.java [hunk 2]
```diff
@@ -102,14 +97,17 @@ public boolean onCreateOptionsMenu(Menu menu) {
...
     private void initExoPlayer() {
-        BandwidthMeter bandwidthMeter = new DefaultBandwidthMeter();
-        TrackSelection.Factory videoTrackSelectionFactory =
-                new AdaptiveTrackSelection.Factory(bandwidthMeter);
-        TrackSelector trackSelector =
-                new DefaultTrackSelector(videoTrackSelectionFactory);
-
-        exoPlayer = ExoPlayerFactory.newSimpleInstance(this, trackSelector);
-        exoPlayerView = ((SimpleExoPlayerView) findViewById(R.id.exoPlayer));
+//        DefaultBandwidthMeter bandwidthMeter = new DefaultBandwidthMeter();
```
Comment (const-cloudinary) [inline sample/src/main/java/com/cloudinary/android/sample/app/ImageActivity.java]: It's not clear what is the purpose of the commented out lines code.

If not needed, please remove

**Review by const-cloudinary — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (adimiz1) [reply .travis.yml:37]: It didnt work at all after bumping the Java version
This fix was taken from Travis forums.

File: sample/src/main/java/com/cloudinary/android/sample/app/ImageActivity.java (hunk 2 above)
Comment (adimiz1) [reply sample/src/main/java/com/cloudinary/android/sample/app/ImageActivity.java]: Deleted, thank you

**Pull request closed without merging**